app/javascript/components/pr-feed.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <ul
+    class="card-pr__list"
+  >
+    <li
+      v-for="pullRequest in pullRequests"
+      :key="pullRequest.id"
+    >
+      <div class="card-pr__orientation">
+        <div class="card-pr">
+          <a class="card-pr__name" v-bind:href="pullRequest.html_url">
```

Comment:
Attribute "bind" should be on a new line         vue/max-attributes-per-line<br>Unexpected 'v-bind' before ':'                   vue/v-bind-style

Reply:
es redundante poner el `v-bind,` se puede poner solamente `:href="link"`